pkg/release/release.go

```diff
@@ -33,3 +42,28 @@ var Image = map[string]string{
 	"pod":                       "quay.io/microshift/pause:" + Base,
 	"service_ca_operator":       "quay.io/microshift/service-ca-operator:" + Base,
 }
+
+func init() {
+	arch_replacer := strings.NewReplacer("amd64", "x86_64", "arm64", "aarch64")
+	arch := arch_replacer.Replace(runtime.GOARCH)
+
+	release_file := "release/release-" + arch + ".json"
+	data, err := embedded.Asset(release_file)
+	if err != nil {
+		// If there is no release file for this architecture, work with the generic specs
```

Comment:
Is it a good time to remove generic specs? Are they any good?

Reply:
In the upstream MicroShift project, this has been our fallback for cases when no equivalent images from OCP exist that we could reference. My assumption is we'll needed when we revive the upstream community.